Input/Core/InputSystem.cs

```diff
@@ -41,6 +55,14 @@ static InputSystem()
 			}
 
 			s_Devices.InitAfterProfiles();
+		    NativeInputDeviceManager.onDeviceDiscovered -= RegisterDevice;
```

Comment:
NativeInputDeviceManager sticks around through multiple InputSystem lifecycles so we want to make sure these callbacks get unregistered. We do this in the constructor since InputSystem has no destructor.